project/AppDependencies.scala

```diff
@@ -6,7 +6,7 @@ object AppDependencies {
   val play26Version = "2.6.20"
   val play27Version = "2.7.4"
 
-  val compileCommon = Seq(
+  val httpVerbsCoreCompileCommon = Seq(
```

Comment:
drop the `httpVerbs` prefix - it's a given?

Reply:
“You should name a variable using the same care with which you name a first-born child.”

Agreed, done.